Sources/Plasma/PubUtilLib/plScene/plRelevanceMgr.cpp

```diff
@@ -122,19 +120,18 @@ void plRelevanceMgr::SetRegionVectors(const hsPoint3 &pos, hsBitVector &regionsI
     if (!inAnyRegion)
     {
         regionsImIn.SetBit(0, true);
-        regionsICareAbout.Set(fRegions.GetCount());
+        regionsICareAbout.Set(fRegions.size());
     }
 }
     
 uint32_t plRelevanceMgr::GetNumRegions() const
 {
-    int i;
+    hsSsize_t i;
 
-    for (i = fRegions.GetCount(); i > 0 && fRegions[i - 1] == nullptr; i--);
+    for (i = fRegions.size(); i > 0 && fRegions[i - 1] == nullptr; i--);
```

Comment:
Actually, in this case it is definitely not more readable...  We're basically taking the size of the vector and subtracting the number of `nullptr` entries at the end.  It could probably be written to be more readable, but not with `std::find` or `std::find_if`.